.eslintrc.js

```diff
@@ -0,0 +1,135 @@
+const OFF = 0;
+const WARNING = 1;
+const ERROR = 2;
```

Comment:
We don't need these -- you can use strings in eslint now: https://github.com/eslint/eslint/blob/master/packages/eslint-config-eslint/default.yml